lib/golomb/golomb.js

```diff
@@ -39,10 +38,14 @@ class Golomb {
    * @constructor
    */
 
-  constructor() {
+  constructor(p,m) {
```

Comment:
```suggestion
  constructor(p, m) {
```

Reply:
actually gonna send you a diff on discrod about capital and lowercase properties....